apps/src/javalab/JavalabView.jsx

```diff
@@ -121,7 +139,10 @@ class JavalabView extends React.Component {
               style={styles.finish}
             />
           </div>
-          <div style={styles.editorAndVisualization}>
+          <div
+            ref={ref => (this.EditorAndVisualization = ref)}
```

Comment:
naming nit: should be `editorAndVisualization`

Reply:
fixed!